### PR TITLE
starting image > drink task

### PR DESCRIPTION
**Pull request by paroma**
lots going on:
- added drink task so the BabbleModel notebook runs end to end
- created drink folder with copies of bike stuff (notebooks were me testing, and drink_model looks at the relevant .npy files)
- changeImageExtractor because it was specific to the bike task, added an argument that defaults to bike task to not break existing scripts
- changed post-processing a bit to account for bad candidates and not throw error
- current Drink task is the"able to reach cup" query, though we have necessary files for the "holding a cup" query as well

### CODE REVIEW

File: tutorials/babble/drink/image_objects.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import numpy as np
```
Comment (bhancock8) [inline tutorials/babble/drink/image_objects.py]: Why are we reintroducing Image_Candidate and BBox objects. Do the ones in context.py that we use for the bike task not work for the drink task?

Comment (paroma) [reply tutorials/babble/drink/image_objects.py]: they should, i just copied most of the files that existed in the bike/ folder so we shouldn't include this second file

File: tutorials/babble/drink/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+from drink_model import DrinkModel
+from image_objects import BBox
```
Comment (bhancock8) [inline tutorials/babble/drink/__init__.py]: We shouldn't use this BBox object anywhere.

Comment (paroma) [reply tutorials/babble/drink/__init__.py]: removing stuff!

**Comment by paroma** [comment]
removed outdated files

**Pull request merged**